src/core/int128.d

```diff
@@ -67,7 +67,14 @@ Cent com(Cent c)
 pure
 Cent neg(Cent c)
 {
-    return inc(com(c)); // ~c + 1
+    if (c.lo == 0)
+        c.hi = -c.hi;
```

Comment:
To properly test `neg`, please add the following test:
```
    const Cm10_0 = Cent(0, -10);
    const Cm10_1 = Cent(-1, -11);
    const Cm10_3 = Cent(-3, -11);

    assert(neg(C10_0) == Cm10_0);
    assert(neg(C10_1) == Cm10_1);
    assert(neg(C10_3) == Cm10_3);
```

Reply:
ok